lib/gui/app/components/safe-webview.js

```diff
@@ -66,7 +66,7 @@ const API_VERSION_PARAM = 'api-version'
  * This version number is exposed to the banner such that it can determine what
  * features are safe to utilize.
  */
-const API_VERSION = 1
+const API_VERSION = 2
```

Comment:
Perhaps it'd be worth adding some comments, clarifying what changes each of the different API_VERSIONs signify?

Reply:
That's a good idea, but it'd grow with every version change, instead maybe we could just have a comment like below.

```
// For history use: git blame -L n
```

Reply:
> but it'd grow with every version change

I guess it depends on how often you expect `API_VERSION` to change...